src/app/report/edit-display/edit-display.component.ts

```diff
@@ -80,12 +81,11 @@ export class EditDisplayComponent {
   rerunReport(): void {
     let reportId: string = this.report.storageId;
     this.httpService.runDisplayReport(reportId, this.currentView.storageName).subscribe((response) => {
-      let element = document.querySelector('#showRerunResult')!;
       if (this.report == response) {
-        element.setAttribute('style', 'background-color: green');
+        this.rerunResultElement.nativeElement.setAttribute('style', 'background-color: green');
```

Comment:
I think it's better to use a class variable called runSuccessful or something and then use ngClass or style.background-color in the html to apply a different background color